main.c

```diff
@@ -53,6 +55,9 @@
 #include "libtcmu_config.h"
 #include "libtcmu_log.h"
 
+#define TCMUR_MIN_OPEN_FD 65536
+#define TCMUR_MAX_OPEN_FD 1048576
```

Comment:
What about making this a tcmu.conf setting? Checkout the libtcmu_config.c code for examples on how to add new settings.